lucene/common-build.xml

```diff
@@ -63,7 +63,7 @@
 
   <fail message="If you pass -Dversion=... to set a release version, it must match &quot;${version.base}&quot;, optionally followed by a suffix (e.g., &quot;-SNAPSHOT&quot;).">
     <condition>
-      <not><matches pattern="^\Q${version.base}\E(|\-.*)$" casesensitive="true" string="${version}"/></not>
+      <not><matches pattern="^\Q${version.base}\E(|.*)$" casesensitive="true" string="${version}"/></not>
     </condition>
```

Comment:
At least there must be a `\b` (boundary) instead of the dash. This would match a version like 1.2.34 as valid for q.w.e base version.